test/auth_aws/test_auth_aws.py

```diff
@@ -54,5 +54,40 @@ def test_connect_uri(self):
             client.get_database().test.find_one()
 
 
+class TestAWSLambdaExamples(unittest.TestCase):
+    def test_shared_client(self):
+        # Start AWS Lambda Example 1
+        from pymongo import MongoClient
+
+        client = MongoClient(host=os.environ["MONGODB_URI"])
+
+        def lambda_handler(event, context):
+            return client.db.command("ping")
+
+        # End AWS Lambda Example 1
+
+    @unittest.skipIf(
+        not os.environ.get("AWS_ACCESS_KEY_ID"), "Must have IAM credentials in " "environment."
```

Comment:
combine the strings: `"Must have IAM credentials in " "environment."`

Also would this test ever actually run in EVG? If not how do we ensure it does?

Reply:
I removed the skip so it always runs now that we are using implicit credentials.